doc/openapi/README.md

```diff
@@ -1,17 +1,19 @@
 # OpenAPI / Swagger Spec Files
 This code generates OpenAPI 3.0 spec based on the [Postman Collection](../postman/device-onvif-camera.postman_collection.json), 
 [Onvif WSDL Schema Files](./ref), [sidecar.yaml](sidecar.yaml), [default camera device profile](../../cmd/res/profiles/camera.yaml),
-[ONVIF Tested Cameras Matrix](../ONVIF-protocol.md#tested-onvif-cameras), and [ONVIF footnotes](../onvif-footnotes.md).
+[ONVIF Tested Cameras Matrix](https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md#tested-onvif-cameras), and [ONVIF footnotes](https://github.com/edgexfoundry/edgex-docs/blob/main/docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/onvif-footnotes.md).
```

Comment:
Should these point to the files on github or the url to the docs site itself?

Reply:
You mean should they point to files in this repo? But were they not all moved to Edgex docs?

Reply:
I mean using this link instead: https://docs.edgexfoundry.org/3.0/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol/#tested-onvif-cameras

Reply:
The reason I pointed them to github is because it helps the developer know where to find the file that needs to be modified.